runbotics/runbotics-orchestrator-ui/src/src-app/Actions/general.actions.ts

```diff
@@ -23,7 +23,7 @@ const getGeneralActions: () => Record<string, IBpmnAction> = () => ({
                                 additionalProperties: {
                                     mainFieldLabel: translate('Process.Details.Modeler.Actions.General.ConsoleLog.Message'),
                                     subFieldLabel: translate('Process.Details.Modeler.Actions.General.ConsoleLog.Variable'),
-                                    isRequired: true,
+                                    isRequired: false,
```

Comment:
why is it no longer required?

Reply:
does console.log without any message make sense? we always want to display some message, usually variable